src/transformers/generation/utils.py

```diff
@@ -1297,6 +1297,28 @@ def _validate_generated_length(self, generation_config, input_ids_length, has_de
                     UserWarning,
                 )
 
+    def _extend_attention_mask(self, model_kwargs: Dict[str, Any], to: int) -> Dict[str, Any]:
```

Comment:
```suggestion
    def _extend_attention_mask(self, model_kwargs: Dict[str, Any], new_mask_length: int) -> Dict[str, Any]:
```
`to` is a common PyTorch operator, so we should avoid using it as a variable (hurts readability). Feel free to pick a different variable name, as long as it is descriptive :)